components/contact/ContactForm.vue

```diff
@@ -768,8 +847,10 @@ export default {
   },
   computed: {
     invalidFields() {
-      return Object.keys(this.$v.contactInfo.$params)
-        .filter(fieldName => this.$v.contactInfo[fieldName].$invalid)
+      return Object.keys(this.$v.contactInfo.$params).filter(
+        // eslint-disable-next-line arrow-parens
```

Comment:
There may be some config drift here with our linters. This does not throw a error for me.